README.md

```diff
@@ -98,7 +102,8 @@ import sys
 MULTIRUN = os.getenv("PDM_MULTIRUN", "0") == "1"
 
 if MULTIRUN:
-    py = f"{sys.version_info[0]}.{sys.version_info[1]}"  # 3.8, 3.9, etc.
+    int_name = os.getenv('PDM_MULTIRUN_CURRENT', '')
+    py = f"{int_name}: {sys.version_info[0]}.{sys.version_info[1]}"  # py38: 3.8, py39: 3.9, etc.
```

Comment:
Maybe show a venv name that is not just the Python version, to make it more explicit?

`build: 3.8, tests: 3.11` etc.

Reply:
```suggestion
    py = f"{int_name}: {sys.version_info[0]}.{sys.version_info[1]}"  # build: 3.8, tests: 3.9, etc.
```